### DISCUSSION TITLE
Api Rules Questions

### DISCUSSION THREAD

**Discussion by baguetteism** [pocketbase/pocketbase#1878] [Q&A] [answered]
I need to return data only rows that contain data like this:

`@collection.example.user_ref` equal to
`@request.auth.user_ref` 

Can do it only set API Rules without filter parameter when fetch data or not 

Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
I'm sorry, I don't understand the question.

Could you elaborate a little more by providing more info about your collection structure(s) and example output (you can export your collections as json from "Admin UI > Settings > Export collections")?


I don't understand what is `user_ref` in the above and why you want to use `@collection.*` filter.
If you haven't already, you can check https://pocketbase.io/docs/api-rules-and-filters/ for general info about the filters and rules syntax.

**Reply by baguetteism** [reply]
My "example" and "users" have 'user_ref' field when authenticated user get list from "example" collection I need to return data only which rows is user_ref field in "example" collection equal to "user_ref" in @request.auth.*

**Reply by ganigeorgiev** [reply]
In this case, for your `example` collection rules you can just use:

```js
@request.auth.user_ref != "" &&
user_ref = @request.auth.user_ref
```

**Reply by baguetteism** [reply]
It's work fine.
Thanks.
